eMarket/app/models.py

```diff
@@ -24,8 +24,13 @@ def __str__(self):
 class Cart(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, limit_choices_to={'user_type': 'buyer'})
     product = models.ForeignKey(Product, on_delete=models.CASCADE)
-    quantity = models.PositiveIntegerField(default=1)
-
-
+    quantity = models.PositiveIntegerField(default=0)
+    def get_total(self):
```

Comment:
Does this work properly? From what I can see it looks like you won't get the correct value if you have more then one type of product in the cart.

Reply:
Never mind I looked at the implementation of it. Consider changing the function name to something more descriptive because it's a bit misleading currently

Reply:
The get_total function

Reply:
Ohh like I should maybe name it "get_price" or something like that?